src/client/delphi_epidata.py

```diff
@@ -88,6 +105,8 @@ def _request(endpoint, params={}):
         """
         try:
             result = Epidata._request_with_retry(endpoint, params)
+            if Epidata.sandbox:
+                return
```

Comment:
```suggestion
```
if ._request_with_retry() returns a proper Response object, you can hopefully remove this?